SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpEventDto.cs

```diff
@@ -58,7 +58,7 @@ public class MpEventDto
         [JsonProperty(PropertyName = "Cancelled")]
         public int Cancelled { get; set; }
 
```

Comment:
would this cause any unintended consequences? I would think this would be a boolean